src/Network/QDisc/Fair.hs

```diff
@@ -15,6 +16,16 @@ import Network.Transport (EndPointAddress)
 import Network.Transport.TCP (QDisc(..))
 import Data.Time.Units (Microsecond)
 
+blockingQDisc :: (EndPointAddress -> IO ()) -> IO (QDisc t)
+blockingQDisc waitForUnblock = do
+    fqd <- newFairQDisc
+    return QDisc {
+          qdiscEnqueue = \addr _ t -> do
+                  waitForUnblock addr
+                  writeFairQDisc fqd t
+        , qdiscDequeue = readFairQDisc fqd
+        }
+
```

Comment:
Seems this definition is not used. Was that unintentional, or should we remove it?